src/Conference/ConferenceCall.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import { Switch, Route, useParams } from "react-router-dom";
+import ConferenceDetails from "./ConferenceDetails";
+import UpdateConference from "./UpdateConference";
+import { getConferenceById } from "../api.js";
+
+const ConferenceCall = () => {
```

Comment:
I'm not quite sure about the name ConferenceCall. In React parlance you'd probably call this a ConferenceContainer.